RecoLocalTracker/SiPixelClusterizer/test/gpuClustering.cu

```diff
@@ -174,88 +158,73 @@ int main(void)
   size_t size32 = n * sizeof(unsigned int);
   size_t size16 = n * sizeof(unsigned short);
   size_t size8 = n * sizeof(uint8_t);
-	 
+
   uint32_t nModules=0;
   cuda::memory::copy(d_moduleStart.get(),&nModules,sizeof(uint32_t));
-  
+
   cuda::memory::copy(d_id.get(), h_id.get(), size16);
   cuda::memory::copy(d_x.get(), h_x.get(), size16);
   cuda::memory::copy(d_y.get(), h_y.get(), size16);
   cuda::memory::copy(d_adc.get(), h_adc.get(), size8);
-  cuda::memory::device::zero(d_debug.get(),size32);  
+  cuda::memory::device::zero(d_debug.get(),size32);
   // Launch CUDA Kernels
-
-  
   int threadsPerBlock = 256;
   int blocksPerGrid = (numElements + threadsPerBlock - 1) / threadsPerBlock;
   std::cout
```

Comment:
remove?

Reply:
this is a test; it is normal to print something!